runtime/src/snapshot_utils/archive_format.rs

```diff
@@ -20,6 +20,7 @@ pub enum ArchiveFormat {
     TarZstd,
     TarLz4,
     Tar,
+    None,
```

Comment:
Here if I don't add None, I will have to make many small changes to make ArchiveFormat optional, because those non-None values do not make sense in the construct-from-file route.  Not sure if it worth the effort to make all those changes.

Reply:
there's some talk on discord about removing the uncompressed options anyway: https://discord.com/channels/428295358100013066/838890116386521088/1050920232945979493

which removes my concern with the naming here.